javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/extension/instrumentation/internal/injection/ExtendedInstrumentationModule.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.extension.instrumentation.internal.injection;
```

Comment:
I'd drop the `injection` from package.

Reply:
Agree, that `injection` actually slipped in there by mistake.